helpers/auth/admin_roles.py

```diff
@@ -19,6 +19,14 @@ def create_rooms_update_delete_office(self, office_id):
         if admin_details.location != location.name:
             raise GraphQLError("You are not authorized to make changes in " + location.name)  # noqa: E501
 
+    def verify_admin_location(self, location_id):
+        admin_details = get_user_from_db()
+        location = Location.query.filter_by(id=location_id).first()
+        if admin_details.location != location.name:
```

Comment:
@RachaelNantale considering that when adding a user, location is added as a string, how can we guarantee that the user location string and location name will always match?

Reply:
Thank you @joshuaocero, So after the brief sync we had I went through the code and discovered that there are two ways of creating a user. 
The Auto creation when one is interacting with the other queries that need a user and the manual creation using the fixture.
When a user is created automatically, the email, name and picture are saved. The location, on the other hand, is gotten from the `andela api` when a request is sent using the email from the email and token.
So the location is not created by us.

Reply:
Noted @RachaelNantale. True that the user location is not created by us but the location name on the Location model is. Kindly add a TODO comment so we remember to refactor this section when we reorganise the User <> Location relationship.

Reply:
Okay @joshuaocero, this has been implemented.